hbogolib/handlereu.py

```diff
@@ -19,6 +19,8 @@
 
 import requests
 
+import re
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [standard import "import re" should be placed before "import requests"](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4579233)